### ISSUE TITLE
Write Rust doc for FirstMsgCommitments.

### ISSUE THREAD

**Issue by gbenattar** [KZen-networks/multi-party-ecdsa#8]
Write Rust doc (https://doc.rust-lang.org/book/first-edition/documentation.html). Mention in the doc papers, math formula and explanation.

See: https://github.com/KZen-networks/multi-party-ecdsa/blob/master/src/protocols/two_party_ecdsa/lindell_2017_keygen/party_one.rs#L51

**Comment by gbenattar** [comment]
Adding more doc, see: https://github.com/KZen-networks/multi-party-ecdsa/issues/11 for references.

**Comment by jeamick** [comment]
hey @gbenattar,

this link (See: https://github.com/KZen-networks/multi-party-ecdsa/blob/master/src/protocols/two_party_ecdsa/lindell_2017_keygen/party_one.rs#L51) is not working...

Here is the link : https://github.com/KZen-networks/multi-party-ecdsa/blob/master/src/protocols/two_party_ecdsa/lindell_2017/party_one.rs#L51

I'll start dig into this issue if it is ok ?

Thanks

JM

**Comment by omershlo** [comment]
awesome

**Comment by gbenattar** [comment]
@jeamick Yes that would be great, I updated slightly the task description. At the end the outcome of it should be a proper Rust doc for this crate. Feel free to split it into small chunk if you want early review from @omershlo. 

Let us know if you need any help, I suppose that you are very familiar with 2P-ECDSA, we have many tasks as a follow up.

Thanks.
